ruby/Gemfile

```diff
@@ -0,0 +1,5 @@
+source "https://rubygems.org"
+
+# See https://github.com/algolia/algoliasearch-client-ruby 
+ gem "algoliasearch", ">=1.26"
+ gem 'dotenv'
```

Comment:
Why we need this?

Reply:
i was wondering if it was useful to use a .env file. But if we keep playground simple we should remove it